src/index.js

```diff
@@ -121,6 +121,13 @@ const getAssetFromKV = async (event, options) => {
   if (response) {
     let headers = new Headers(response.headers)
     headers.set('CF-Cache-Status', 'HIT')
+    if (options.cacheControl.browserTTL === 0 || options.cacheControl.browserTTL) {
```

Comment:
I think we should separate this boolean check to its own function that returns a boolean, and it should also return false if `options.cacheControl.browserTTL` is negative

Reply:
so like:
```
const toSetBrowserTTL = cacheControl => cacheControl.browserTTL === 0 || cacheControl.browserTTL
```

Reply:
readability and also so we don't repeat ourselves, this code is used twice - generally that should be a separate function imo. 

```js
const isValidBrowserTTL = browserTTL => {
  let ttl = parseInt(browserTTL, 10)
  if (typeof ttl !== "number" || ttl < 0) {
    return false
  }
  return true
}
```